projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/jupyter_notebook.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import nbformat
+from nbconvert.preprocessors import ClearOutputPreprocessor
+
+
+class NotebookOutputCleaner:
```

Comment:
This class can simply be a function in vdk_ui.py.

Reply:
done